src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
```

Comment:
Add empty lines between sibling blocks of HTML to add some 'air' and simplify reading. There should be an empty line between the closing `</div>` of one `.stars` block and the opening `<div>` of the next `.stars` block.